languagetool-language-modules/eo/src/main/resources/org/languagetool/rules/eo/grammar.xml

```diff
@@ -4047,6 +4080,16 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         <example correction="simplan|facilan">Li donis <marker>malkomplikan</marker> respondon.</example>
       </rule>
     </rulegroup>
+    <rulegroup id="HUMIDA" name="humida → malseka">
+      <rule>
+        <pattern>
+          <marker><token inflected="yes" regexp="yes">(ne|mal)?humid((umi|igi|iĝi)|(a|o|eco|ega|eta)j?n?)</token></marker>
+        </pattern>
+        <message>La vorto ‘humida’ estas ne estas oficiala esperanta vorto, kaj la vorto ‘malseka’ normale signifas ekzakte la saman. Ĉu vi intencis ‘<suggestion><match no="1" regexp_match="humid" regexp_replace="malsek"/></suggestion>’?</message>
```

Comment:
General remark: instead of quotes like 'humida', I suggest to use «humida» to be consistent with the other existing rules.

I'd also suggest to write it «\1» since the word is not always humida, it could be humideco for example, so saying humida may be confusing.

But I don't really agree with this rule.  Humida is in PIV and ReVo  (with a slightly different meaning than malseka).  According to ReVo  http://www.reta-vortaro.de/revo/art/humid.html#humid.0a  the word humida is should only avoided outside of a scientific context.  But that advice is quite fuzzy.

I even find «humida» in the dictionary of the Acdemy of Esperanto: http://www.akademio-de-esperanto.org/akademia_vortaro/index.html
So I don't think we can say that it's not an official Esperanto word.  I'd suggest to either:
* remove this rule
* or add context to indicate it as an error if we can find pattern where it's clearly a mistake (with a link to ReVo or perhaps other source of information about it).